bindata/network/ovn-kubernetes/common/default-net-annotation-policy.yaml

```diff
@@ -0,0 +1,38 @@
+{{if .OVN_PRE_CONF_UDN_ADDR_ENABLE}}
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: default-network-annotation
+spec:
+  matchConstraints:
+    resourceRules:
+      - apiGroups:   [""]
+        apiVersions: ["v1"]
+        operations:  ["UPDATE"]
+        resources:   ["pods"]
+  failurePolicy: Fail
+  validations:
+    # Prevent any changes to the default-network annotation after pod creation:
+    # - If annotation exists in old pod: new pod must have same annotation with identical value
+    # - If annotation doesn't exist in old pod: new pod must also not have it
+    - expression: >
+        !has(object.metadata.annotations) || !has(oldObject.metadata.annotations) ||
```

Comment:
slight change here that is missing u/s as noted by @kyrtapz already in his comment on the PR. This fix will also go upstream.